src/query/query_executor.h

```diff
@@ -967,6 +967,7 @@ struct xasl_node
   const char *query_alias;
   int dbval_cnt;		/* number of host variables in this XASL */
   bool iscan_oid_order;
+  int max_iterations;
```

Comment:
Please confirm `xasl->max_iterations` has no issue regarding reusing xasl node, i.e, xclone.
I'm saying a case that an XASL clone is cached and the parameter is changed. It is defined as `(PRM_FOR_SERVER | PRM_USER_CHANGE | PRM_FOR_SESSION | PRM_FOR_CLIENT)`.
It seems the current code is correct but I'd like to have your confirmation.